### ISSUE TITLE
Refine GUI look and feel?

### ISSUE THREAD

**Issue by Splitwirez** [waliarubal/Jaya#62]
Currently, Jaya uses the default Avalonia theme as its default theme. Personally, I think we could do better. Normally my taste is stuck in 2003, but I decided to try to put that aside, and created this mockup:
![](https://i.imgur.com/Y6AK7SK.png)
I was aiming for a more refined version of what Windows 10's File Explorer seems to be going for. Thoughts?

(also no, I'm *not* suggesting we steal Windows 10's icons, I just hadn't yet figured out what to do instead)

**Comment by giorgiozoppi** [comment]
Install something like expandrive or cyberduck and see the differencies.  I miss something but i don't know.

**Comment by Splitwirez** [comment]
Er...CyberDuck seems to just use standard Windows visual styles, and ExpanDrive literally inserts itself into File Explorer...are you implying we should try to mimic Windows 10's appearance *exactly*?

**Comment by giorgiozoppi** [comment]
No, we should evaluate the competitors and see what strategy fits better to us. So i will ask you a reserarch on different file manager and produce a kind of presentation. Does it make sense?

**Comment by waliarubal** [comment]
Yeah, this makes sense but as of now we are just 3 people on this project unless we have someone willing to conduct extensive market research to rule out good features from file managers in market which we shall also provide. The initial inspiration for me was Windows 10 file manager as of in UI and feature set it provides, so we have been trying to take in research what MS has already done for us.

**Comment by Splitwirez** [comment]
True...but this particular issue post isn't about features. It's just about the app's appearance.

Do you two want Jaya to be visually pleasing, or is that just me?

**Comment by waliarubal** [comment]
Well I want it to be visually pleasing just like Windows 10 file explorer. People do care about UI and UX. Well, that's my take of it.

**Comment by Splitwirez** [comment]
Hm...I'd like if we could all agree on a single design to aim for, as that would allow me to get started and know exactly what I'm working towards. Does that make sense to you?

**Comment by Splitwirez** [comment]
An alternative proposed style was raised on Gitter: ![](https://docs.microsoft.com/it-it/azure/media/vs-azure-tools-storage-manage-with-storage-explorer/overview.png)

I'll be preparing a Jaya mockup based on this look fairly soon.

**Comment by Splitwirez** [comment]
![](https://i.imgur.com/tg1vZFZ.png) corresponding mockup, as promised.

**Comment by giorgiozoppi** [comment]
@Splitwirez this is far better. I will put optional recent files in order to have more space.

**Comment by waliarubal** [comment]
@Splitwirez this seems to me perfect solution for presenting tabbed interface. We may put up some minor tweaks, excellent for starting things up. Nicely done, what about the dark theme part of it?

Dark is the new blew these days!

**Comment by waliarubal** [comment]
@Splitwirez  and what about the toolbar interface which we are already providing? Will that be included?

**Comment by Splitwirez** [comment]
> @Splitwirez this seems to me perfect solution for presenting tabbed interface. We may put up some minor tweaks, excellent for starting things up. Nicely done, what about the dark theme part of it?

Excellent. Now that you've approved the light theme, I'll get started on a corresponding dark variant.


> @Splitwirez and what about the toolbar interface which we are already providing? Will that be included?

I definitely think we should keep the Toolbar option...though I think there's ample room for improvement over Jaya's existing Toolbar UI. I guess that's another mockup for me to make.

**Comment by Splitwirez** [comment]
![](https://i.imgur.com/d0JwkLi.png)

How does this look for the dark variant?

**Comment by waliarubal** [comment]
This seems OK to me, what about yourself @giorgiozoppi?

**Comment by giorgiozoppi** [comment]
@waliarubal ok

**Comment by Splitwirez** [comment]
Alrighty, here's the winning pair of mockups:
![](https://i.imgur.com/Y6AK7SK.png)
![](https://i.imgur.com/1hlWaMW.png)
I'll be starting work on implementing this look now.
